ui/src/components/Publication/SimpleResult/index.tsx

```diff
@@ -26,6 +27,9 @@ const SimpleResult: React.FC<Props> = (props): React.ReactElement => (
                 >
                     {Helpers.formatStatus(props.publication.currentStatus)}
                 </span>
+                {props.user.id === props.publication.createdBy && (
+                    <span className="mr-2 leading-3 text-green-700 dark:text-green-300">( Corresponding Author )</span>
```

Comment:
![image](https://user-images.githubusercontent.com/48569671/217308574-54bf671f-8014-4f23-82b8-7f2ef5ad9a6b.png)

the text inside those small boxes is overflowing on mobile devices

Reply:
Should look like this on mobile now

![3978cedd7fef435cd6851c537daa2d55](https://user-images.githubusercontent.com/69204118/217509094-85c340d4-4f77-40c8-8b30-7ae5bc3fb183.png)

Reply:
it's not overflowing anymore but still looks odd on some  breakpoints
![image](https://user-images.githubusercontent.com/48569671/218048376-1967b339-0e07-447b-865d-ac85146fe9f2.png)

DOI is not aligned to the rest of the elements in there
![image](https://user-images.githubusercontent.com/48569671/218048666-8c976a3e-787c-4b67-be39-f266b045d302.png)

![image](https://user-images.githubusercontent.com/48569671/218048709-5179fd6d-15ea-4f23-abb9-ce7c43e2c967.png)

'Draft' should stay on the first line, next to edit icon. The rest of the elements should wrap on the next line
![image](https://user-images.githubusercontent.com/48569671/218048930-8d2be385-70fd-483d-ac3d-60aac5b54368.png)